src/site/xdoc/download_configuration.xml

```diff
@@ -145,36 +145,6 @@ limitations under the License.
         </table>
       </subsection>
     </section>
-    <section name="Apache Commons Configuration 1.10 (old 1.x version)">
-      <subsection name="Binaries">
-        <table>
-          <tr>
-              <td><a href="[preferred]/commons/configuration/binaries/commons-configuration-1.10-bin.tar.gz">commons-configuration-1.10-bin.tar.gz</a></td>
```

Comment:
All this stuff is generated based on POM properties so it will show up again unless you edit pom.xml.

Reply:
ah, removed the `commons.release.2`, LMK if I missed anything else (or if there's an easy way for me to check for myself)